vignettes/adrg.qmd

```diff
@@ -0,0 +1,785 @@
+|                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| The package installation procedure may take a few minutes or longer depending on internet bandwidth.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
++-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+5.  Upon completion of installing packages using {renv}, please restart your R session.
+
+6.  Install the {pilot3utils} package running the code below.
```

Comment:
Choose Package Pane

<img width="735" alt="image" src="https://github.com/RConsortium/submissions-pilot3-adam/assets/10111024/45edee0b-e4d3-414c-92a7-f908c39cebcb">

Click Intall From and change from zip file
<img width="592" alt="image" src="https://github.com/RConsortium/submissions-pilot3-adam/assets/10111024/629c8fd7-bfc7-4c9e-959f-e888f1c7447d">

Select the zip file for pilots3utils
<img width="1045" alt="image" src="https://github.com/RConsortium/submissions-pilot3-adam/assets/10111024/84b909da-0523-4c96-948f-f0c24407e376">

<img width="458" alt="image" src="https://github.com/RConsortium/submissions-pilot3-adam/assets/10111024/f6a06b0d-f8d2-477e-9afc-f26c3e2904fd">

Click install

Reply:
I didn't have time to record a video - ust did some screenshots.  But the process is pretty simple to install the zip file

Reply:
No worries, @bms63 .  Thanks for this.  I've added the steps in words without the image captures for now.